package.json

```diff
@@ -35,7 +35,8 @@
     "storybook": "start-storybook -p 6006",
     "test": "npx jest",
     "test:integration": "./scripts/test-integration",
-    "test:watch": "jest --watch"
+    "test:watch": "jest --watch",
+    "chromatic": "npx chromatic"
```

Comment:
Why do you use `npx` here if you list it as a dependency?

Reply:
why indeed